### PR TITLE
Fix result formats of ain_validateAppName() and ain_checkProtocolVersion()

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix result format of ain_validateAppName() json rpc API
- Let ain_checkProtocolVersion() json rpc API return boolean result
- Check empty tx_list in ain_sendSignedTransactionBatch() json rpc API
- Make json rpc APIs always return result

Related PRs:
- https://github.com/ainblockchain/ain-js/pull/82

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1053

### CODE REVIEW

File: common/version-util.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ class VersionUtil {
       res.status(200)
         .set('Content-Type', 'application/json')
         .send({
+          result: null,
```
Comment (liayoo) [inline common/version-util.js:53]: question) is there a reason why we use null here and false for other APIs?

Comment (platfowner) [reply common/version-util.js:53]: This is a proto version validator applied for all json-rpc requests, so it doesn't know what's the expected type of 'result' (i.e., whether it's boolean or not).

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review. Now merging.. @minsulee2 @liayoo

**Pull request merged**